src/shared/navgen/nav.cpp

```diff
@@ -943,11 +945,27 @@ void NavmeshGenerator::StartGeneration( class_t species )
 	d_->tw = ( gw + ts - 1 ) / ts;
 	d_->th = ( gh + ts - 1 ) / ts;
 
+	float climb = config_.stepSize;
+	if ( sg_botAutojump.Get() > 0.f )
```

Comment:
The config file is merged so you can store the value of this in the config and customize it per map

Reply:
done.